Encrypter/Rsa/Encrypted.ts

```diff
@@ -0,0 +1,23 @@
+export interface Encrypted {
+	key?: string
+	value: string
+}
+
+export namespace Encrypted {
+	export function is(value: any | Encrypted): value is Encrypted {
+		return (
+			typeof value == "object" &&
+			(value.key == undefined || typeof value.key == "string") &&
+			typeof value.value == "string"
+		)
+	}
+	export function stringify(encrypted: Encrypted): string {
+		encrypted.key = encrypted.key && encrypted.key.length != 4 ? encrypted.key.slice(-2) : encrypted.key
```

Comment:
Can you make this easier to understand, this is a spot were people is going to get confused in the future.